src/shared_modules/content_manager/tests/unit/offlineDownloader_test.cpp

```diff
@@ -132,52 +136,10 @@ TEST_F(OfflineDownloaderTest, SkipFileProcessing)
     m_spUpdaterContext.reset(new UpdaterContext());
     m_spUpdaterContext->spUpdaterBaseContext = m_spUpdaterBaseContext;
 
-    // No paths are expected.
-    expectedData.at("paths").clear();
-
     ASSERT_NO_THROW(OfflineDownloader(HTTPRequest::instance()).handleRequest(m_spUpdaterContext));
     EXPECT_EQ(m_spUpdaterContext->data, expectedData);
 }
 
-/**
- * @brief Tests two downloads one after the other. The file from the second download is different from the prior so the
- * output file gets overrided.
- *
- */
-TEST_F(OfflineDownloaderTest, TwoFileDownloadsOverrideOutput)
```

Comment:
question: This test needs to be removed, if there is a possibility to fix it instead of removing it it would be ideal, as the functionality should still work.

Reply:
Reverted and adapted!